0-SCore/Scripts/Entities/EntityEnemySDX.cs

```diff
@@ -28,12 +28,11 @@
 /// </code>
 /// </example>
 /// </summary>
-public class EntityEnemySDX : EntityEnemy, IEntityOrderReceiverSDX
+public class EntityEnemySDX : EntityHuman, IEntityOrderReceiverSDX
```

Comment:
It might be necessary to rethink this. If we figure out how to descend from `EntityBandit` then zombies will attack these types. We can't put `EntityHuman` as a zombie attack target, since zombies themselves descend from `EntityHuman` so they'll attack each other.